lib/smart_answer/calculators/landlord_immigration_check_calculator.rb

```diff
@@ -19,7 +18,7 @@ module LandlordImmigrationCheck
     )
 
     def self.valid_postcode(postcode)
```

Comment:
I have it on my todo list (along with other improvements to that method's implementation), but I'd prefer to make those changes outside this PR.